docs/basics/overview.md

```diff
@@ -0,0 +1,232 @@
+# Conceptual Overview: How GenSBI is Structured
+
+This page explains the core concepts and architecture of GenSBI to help you understand how the different components work together.
+
+## High-Level Architecture
+
+GenSBI is organized around three main abstractions:
+
+```{mermaid}
+graph TB
+    subgraph Pipeline["Pipeline<br/>(High-level orchestration: training, validation, etc.)"]
+        subgraph Model["Model<br/>(Neural network architecture: Flux1, etc.)"]
+            Wrapper["Model Wrapper<br/>(Adds time/noise handling logic)"]
+        end
+        FlowDiff["Flow/Diffusion<br/>(Loss function, ODE solver, paths)"]
+    end
+    
+    style Pipeline fill:#e1f5ff,stroke:#333,stroke-width:2px
+    style Model fill:#fff4e6,stroke:#333,stroke-width:2px
+    style Wrapper fill:#f0f0f0,stroke:#333,stroke-width:1px
+    style FlowDiff fill:#fff4e6,stroke:#333,stroke-width:2px
+```
+
+## Core Concepts
+
+### 1. Models
+
+**Models** are the neural network architectures that learn to approximate posterior distributions. They are standard Flax NNX modules.
+
+GenSBI provides three main model architectures:
+
+- **Flux1**: A double-stream transformer using Rotary Position Embeddings (RoPE). Best for high-dimensional problems.
+- **Simformer**: A single-stream transformer that explicitly embeds variable IDs. Best for low-dimensional problems.
+- **Flux1Joint**: A single-stream variant of Flux1 for explicit joint modeling. Good for likelihood-dominated problems.
+
+**Example:**
+```python
+from gensbi.models.flux1 import Flux1, Flux1Params
+from flax import nnx
+
+params = Flux1Params(
+    in_channels=1,
+    num_heads=8,
+    depth=12,
+    depth_single_blocks=24,
+    axes_dim=[obs_dim],
+    rngs=nnx.Rngs(0),
+    obs_dim=3,
+    cond_dim=5,
+)
+
+model = Flux1(params)
+```
+
+### 2. Model Wrappers
+
+**Model Wrappers** provide a standard interface for models to be used by ODE/SDE solvers during sampling. They standardize how models are called and provide methods for computing the vector field and divergence needed for numerical integration.
+
+Three types of wrappers exist:
+
+- **Unconditional**: For unconditional density estimation
+- **Conditional**: For conditional inference (standard SBI: estimate θ given x)
+- **Joint**: For joint inference (estimate multiple variables simultaneously)
+
+The wrapper provides:
+- Standardized calling interface for solvers
+- `get_vector_field()` method for ODE integration
```

Comment:
this is used both for ODE and SDE solution, respectively in the case of Flow and Diffusion models